packages/vuetify/src/util/colorUtils.ts

```diff
@@ -189,6 +219,18 @@ export function parseHex (hex: string): Hex {
   return `#${hex}`.toUpperCase().substr(0, 9)
 }
 
+export function parseGradient (
+  gradient: string,
+  colors: Record<string, Record<string, string>>,
+  currentTheme: Partial<VuetifyThemeVariant>,
+) {
+  return gradient.replace(/([a-z]+(\s[a-z]+-[1-5])?)(?=$|,)/gi, x => {
+    return classToHex(x, colors, currentTheme) || x
+  }).replace(/(?<=rgba\()#(([0-9a-f]{3}){1,2})(?=,)/gi, x => {
```

Comment:
this doesn't seem to support `#1234` and `#11223344`

Reply:
Btw I noticed that when you pass `#1234` to the `parseHex` function in colorUtils it returns `#123400FF` instead of `#11223344`.

Reply:
And when I fixed it these tests fail:
![Screenshot from 2020-04-06 12-58-52](https://user-images.githubusercontent.com/35876422/78552223-643c5080-7807-11ea-993c-32473e5f0590.png)

Reply:
what is your fix for this?

Reply:
```
if (hex.length === 3 || hex.length === 4) {
  hex = hex.split('').map(x => x + x).join('')
}
````

Reply:
fo `#00gi` rgb should be now `#0000FF` and rgba `#0000FFFF` (because `#00gi` is first changed to `#00FF` and this one to `#0000FFFF`), so you need to update test (values in `toEqual(...)`

Reply:
Okay, thanks